### PR TITLE
Enable osx/arm

### PR DESCRIPTION
**Pull request by jschueller**
@conda-forge-admin please rerender

### CODE REVIEW

**Comment by jschueller** [comment]
@conda-forge/scalapack any clue why the mpi detection fails ?

**Comment by dalcinl** [comment]
@jschueller No way to tell what's going on. Maybe the Open MPI package is somehow broken? Perhaps using `cmake --trace --debug-output ...` could provide a hint.

File: recipe/build.sh [hunk 1]
```diff
@@ -8,12 +8,18 @@ if [[ "$target_platform" == "osx-64" ]]; then
...
+  # https://github.com/Reference-ScaLAPACK/scalapack/issues/21
+  export FFLAGS="${FFLAGS} -fallow-argument-mismatch"
+fi
+
 mkdir build && cd build
-cmake \
+cmake ${CMAKE_ARGS} \
     $EXTRA_CMAKE \
     -DBUILD_SHARED_LIBS=ON \
-    -DCMAKE_INSTALL_PREFIX="$PREFIX" \
```
Comment (dalcinl) [inline recipe/build.sh]: Why is this line gone?

**Comment by Tobias-Fischer** [comment]
I was able to build it locally (natively) with the following changes, and uploaded the resulting packages to http://anaconda.org/TobiasRobotics/
```diff
diff --git a/recipe/build.sh b/recipe/build.sh
index 2f920f0..69d5d02 100644
--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -8,11 +8,11 @@ if [[ "$target_platform" == "osx-64" ]]; then
   fi
 fi
 
-if [[ "$CONDA_BUILD_CROSS_COMPILATION" == "1" ]]; then
+# if [[ "$CONDA_BUILD_CROSS_COMPILATION" == "1" ]]; then
   EXTRA_CMAKE="-DCDEFS=Add_"
   # https://github.com/Reference-ScaLAPACK/scalapack/issues/21
   export FFLAGS="${FFLAGS} -fallow-argument-mismatch"
-fi
+# fi
 
 mkdir build && cd build
 cmake ${CMAKE_ARGS} \
diff --git a/recipe/meta.yaml b/recipe/meta.yaml
index 5069c37..4170512 100644
--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -28,6 +28,7 @@ requirements:
   host:
     - libblas
     - liblapack
+    - zstd
     - {{ mpi }}
   run:
     - {{ mpi }}
```

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin please rerender

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Pull request closed without merging**